flow/raster_cache_unittests.cc

```diff
@@ -473,6 +482,55 @@ TEST(RasterCache, NestedOpCountMetricUsedForDisplayList) {
   ASSERT_TRUE(cache.Draw(*display_list, dummy_canvas));
 }
 
+TEST(RasterCache, NaiveComplexityScoringDisplayList) {
+  size_t threshold = 1;
+  flutter::RasterCache cache(threshold);
+
+  SkMatrix matrix = SkMatrix::I();
+
+  // Five raster ops will not be cached
+  auto display_list = GetSampleDisplayList(5);
+  ASSERT_EQ(display_list->op_count(), 5);
+  ASSERT_EQ(display_list->complexity_score(), 5u);
+  ASSERT_FALSE(display_list->should_be_cached());
```

Comment:
What will happen with this test case when there are multiple complexity calculators?

Reply:
The test cases will need to be updated. I think that it'd be reasonable to land tests for the functionality that's there right now, then modify/extend them in the future for the additional cases added later.